test.js

```diff
@@ -50,26 +50,38 @@ test('is mobile', function (t) {
 
 describe('ua-bruteforce', function () {
   const limit = 300
-  const checks = {
-    mobile: { result: true },
-    tablet: { result: true, tablet: true },
-    desktop: { result: false }
-  }
-  const testCases = Object.entries(checks).reduce(
-    (cases, [deviceCategory, { result, tablet }]) => [
-      ...cases,
-      ...new Array(limit).fill().map(() => ({
-        ua: new UserAgent({ deviceCategory }).toString(),
-        result,
-        tablet
-      }))
-    ],
+  const checks = [
+    ['mobile', true],
+    ['tablet', true, { tablet: true }],
+    ['desktop', false]
```

Comment:
ok, I think just one last change, could you refactor this data structure to be an array of objects, like `[{ family: 'mobile', tablet: false, ...}]`? This way it's easier to read

Reply:
np